lesson_05/WazeAround/README.md

```diff
@@ -0,0 +1,8 @@
+## User Story 
+### AmiyahJo 
+
+- As a user navigating through a city I don't know , I want Waze to give me a heads up about things like accidents , a certain road has closed down , or hazards on the road so that I feel more at ease with deciding which route to chose. 
```

Comment:
The phrase `user navigating through a city I don't know` should be reduced to the name of a role in the system, like `driver`.

As a driver, I want Waze to give me a heads up about things like accidents...

Reply:
Fixed! Now it's more brief.